app/npm-debug.log

```diff
@@ -0,0 +1,45 @@
+0 info it worked if it ends with ok
```

Comment:
This file shouldn't be checked in. Can you add it to the `.gitignore`, and remove it in another commit?

Reply:
Ok, will add it ``` .gitignore```